src/System.ComponentModel.Annotations/tests/PhoneAttributeTests.cs

```diff
@@ -24,6 +24,21 @@ protected override IEnumerable<TestCase> ValidValues()
             yield return new TestCase(new PhoneAttribute(), "425-555-1212 ext.123");
             yield return new TestCase(new PhoneAttribute(), "425-555-1212 ext. 123");
             yield return new TestCase(new PhoneAttribute(), "1");
+
+            // Certain invalid phone numbers are reported as valid with .NET core.
+            // The full .NET framework considers them invalid. This is likely a bug
+            // in .NET core.
+            if (!PlatformDetection.IsFullFramework)
```

Comment:
I think In the past we've just commented out the test and added a reference to the issue. I can do that if it helps - would also avoid us having to do a bunch of code movement.